server/server.js

```diff
@@ -177,12 +176,18 @@ function sendCorsHeaders(req : ExpressRequest, res : ExpressResponse) {
     if (corsRequestMethod) {
         res.header(HTTP_HEADER.ACCESS_CONTROL_ALLOW_METHODS, corsRequestMethod);
     }
+
+    res.header(HTTP_HEADER.ACCESS_CONTROL_ALLOW_CREDENTIALS, 'true');
 }
 
 export function expressEndpoint({ uri = '/', logger = defaultLogger, enableCors = false } : ExpressEndpointOptions = {}) : mixed {
 
     // $FlowFixMe
     const app = require('express')();
+    const bodyParser = require('body-parser');
+    
+    app.use(bodyParser.text({ type: 'text/plain' }));
```

Comment:
Does this parse text/plain as json though?

Reply:
It parses as string, but was thinking that this occurs when using `sendBeacon` and we pass `JSON.stringify(json)` as the body so it would be good.

Reply:
We probably need to parse the json to be able to destructure and fire off the individual events from the payload